### DISCUSSION TITLE
In BtcPay there should be an feature or provision to know the payout status (for invoice refund process) real time using webhook or some notification feature

### DISCUSSION THREAD

**Discussion by aditya-dhand** [btcpayserver/btcpayserver#2528] [Technical Support]
There should be an automatic refund process for invoices instead of manually refunding using pull payments currently.Payout status should also been notified real-time to merchants using some webhook or Notification feature.So we can implement refund process using API and integrate in our system

**Comment by thiagolcks** [comment]
Yeah, it would be really important to automatize this process. Today we are having to manually set the refund in two systems (BtcPayServer and our control system)
